addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -2832,12 +2832,13 @@ export class OdooEditor extends EventTarget {
         }
 
         const block = this.options.getPowerboxElement();
+        const currentNode = getSelection().focusNode;
         if (block) {
             this._makeHint(block, this.options._t('Type "/" for commands'), true);
         }
 
         // placeholder hint
-        if (this.editable.textContent === '' && this.options.placeholder) {
+        if (this.editable.textContent === '' && this.options.placeholder && (currentNode.nodeName != 'P' && currentNode.parentElement.nodeName != 'P') ) {
```

Comment:
You should use `!==`. It's unlikely to ever make a concrete difference in this case but at least for coherence so we never have to rack our brains with the two different kinds of equality in Javascript.